src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -263,6 +263,10 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
     }
     if (valid) {
       const actualParties = HearingActualsViewEditPartiesComponent.toActualParties(parties);
+      const localHearingOutcome = {
+        ...this.hearingActualsMainModel.hearingActuals.hearingOutcome,
+        hearingType: this.hearingActualsMainModel.hearingPlanned.plannedHearingType
```

Comment:
it should only load the hearingPlanned.plannedHearingType or overide the hearingType when the hearingActuals.hearingOutcome.hearingType is not existing.